packages/synapse-react-client/src/components/EntityFinder/tree/VirtualizedTree.tsx

```diff
@@ -402,33 +418,29 @@ export function getTreeWalkerFunction(
       // Yielding `undefined` indicates to react-vtree that we've finished iterating children from the last time we were in the loop.
       const parentMeta = yield
 
```

Comment:
This generator tells `react-vtree` how to render the tree. I made small changes to the logic/flow to try to make it a bit easier to follow.